README.md

```diff
@@ -113,12 +122,31 @@ To setup a local VTN, we have provided a script and a custom agent configuration
     pip install openleadr
     ```
 
-1. At the top level of this project, run the VTN server in the foreground so that we can observe logs:
+1. At the top level of this repo, run the VTN server in the foreground so that we can observe logs:
 
     ```shell
     python utils/vtn.py
     ```
 
+    You should see the following when you run the VTN:
```

Comment:
How do I change the port....I have 8080 mapped to something else?

Reply:
I just added the option to configure the port for the VTN using an environment variable, `VTN_PORT`.